Tribler/community/multichain/database.py

```diff
@@ -145,6 +145,9 @@ def get_by_hash(self, hash):
         :param hash: The hash of the block that needs to be retrieved.
         :return: The block that was requested or None
         """
+        if hash is None:
```

Comment:
@whirm I some scenario's the hash argument is in itself the result of a query, for example: 
https://github.com/Tribler/tribler/blob/devel/Tribler/community/multichain/database.py#L126
Of course the if None statement could be placed there, but I think this is both safer and more elegant.